src/Pickles/Pickles/FeatureParser.cs

```diff
@@ -90,5 +92,18 @@ private string DetermineLanguage()
             }
             return language;
         }
+
+        private Feature RemoveFeatureWithExcludeTags(Feature result)
+        {
+            if (result.Tags.Any(t => t.Equals($"@{configuration.ExcludeTags}", StringComparison.InvariantCultureIgnoreCase)))
+                return null;
+
+            var wantedFeatures = result.FeatureElements.Where(fe => fe.Tags.All(t => !t.Equals($"@{configuration.ExcludeTags}", StringComparison.InvariantCultureIgnoreCase))).ToList();
```

Comment:
Could you perhaps add a test that shows that both scenarios with lowercase and mixed case versions of the exclude tag are being excluded?